### DISCUSSION TITLE
Path reference problem!

### DISCUSSION THREAD

**Discussion by Dong-Qiu-Long** [facebook/docusaurus#8937] [General]
<h2>The link in green cannot be accessed</h2>
<img width="1290" alt="image" src="https://user-images.githubusercontent.com/70185294/234828678-9013cb69-d063-4964-a3af-7275f97ecf2b.png">
<h2>It's compiled like this。No version number</h2>
<img width="1534" alt="image" src="https://user-images.githubusercontent.com/70185294/234829037-d844d4ff-d71c-4172-b94c-3f37c1280167.png">
<h2>It is strange that</h2>
<h2>This one down here does</h2>
<img width="1102" alt="image" src="https://user-images.githubusercontent.com/70185294/234829403-0784efdf-5f3d-46fa-85c7-3f26ec05acb1.png">
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/70185294/234829801-190c3316-c74b-4774-bdb1-1c59bc845fad.png">
<h2>These two are also accessible</h2>
<img width="1190" alt="image" src="https://user-images.githubusercontent.com/70185294/234830754-dce86fec-e846-4206-a813-d62870ae14f5.png">

**Comment by Dong-Qiu-Long** [comment]
<h1>You can't access files in the sibling directory</h1>

**Comment by Dong-Qiu-Long** [comment]
<h1>The root path cannot be used</h1>

**Comment by Dong-Qiu-Long** [comment]
<img width="967" alt="image" src="https://user-images.githubusercontent.com/70185294/234831513-8cc650f7-cb31-4dd8-84bd-ea9151ea95e7.png">

**Comment by Dong-Qiu-Long** [comment]
<img width="412" alt="image" src="https://user-images.githubusercontent.com/70185294/234831686-031445a3-6628-4f94-984c-3bf4d418152f.png">

**Comment by slorber** [comment]
It's really hard for me to follow this discussion. 
Please create a small reproduction of your problem if you want help.

**Reply by Dong-Qiu-Long** [reply]
The 2.0 version number is missing

**Reply by slorber** [reply]
The repro is missing

**Reply by Dong-Qiu-Long** [reply]
<img width="1341" alt="image" src="https://user-images.githubusercontent.com/70185294/234850941-5b9f5b10-271f-4e84-8a0c-4c7530844c17.png">

**Reply by Dong-Qiu-Long** [reply]
The file is pure but not accessible

**Reply by Dong-Qiu-Long** [reply]
<img width="1627" alt="image" src="https://user-images.githubusercontent.com/70185294/235041114-4eb3106d-eb2c-413f-8fe5-bacfdd679140.png">
<img width="1347" alt="image" src="https://user-images.githubusercontent.com/70185294/235041228-d07f0b3f-d7e2-4e6d-8c7d-a709e18871b6.png">
Imported adjacent folder path version number is lost

**Comment by Dong-Qiu-Long** [comment]
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/70185294/234833607-bc023a90-2ee5-4814-9f94-692ba18a0590.png">
<h2>There is no version number after compilation</h2>

**Reply by Dong-Qiu-Long** [reply]
<img width="1582" alt="image" src="https://user-images.githubusercontent.com/70185294/234834052-759dc64d-aead-4fca-ad05-5f2039624719.png">
<h2>The 2.0 version number is missing</h2>

**Comment by Dong-Qiu-Long** [comment]
<img width="1121" alt="image" src="https://user-images.githubusercontent.com/70185294/234834841-1b864208-3c69-4664-a101-9f02a276ae29.png">
<img width="1478" alt="image" src="https://user-images.githubusercontent.com/70185294/234835356-1fa5cd75-f3bc-4646-9f4b-2e26548a3767.png">

<h2>this is correct</h2>
